src/templates/PartnerTemplate.jsx

```diff
@@ -78,9 +81,18 @@ const PartnerTemplateWrapper = styled.div`
   .partner-content: {
     margin: 0 0.5em;
   }
-
+  .description {
+    img{ width: 300px;}
+  }
+  .vacatures {
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
   .partner-joblist {
-    margin: 0px 0.5em;
+    display: grid;
```

Comment:
Somebody should put this in the contributing guidelines, but we decided to use Gatsby's `<Grid>` for homogeneous grids, because they are responsive out of the box.

Reply:
In the progress of changing all homogeneous ones.